canonical-web-and-design/charmhub.io
# Update navigation about page

## Done

Update navigation about page
Update copy docs change /about page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045/about
- Updated navigation to match https://docs.google.com/document/d/1fXDYkfejlIiB-No_qwf-EMA-LTXwK2rZ5hzu1po50p4/edit
- Added new section from [copy doc](https://docs.google.com/document/d/1UuNVajNHcWcF9d-fSnPFl8Zi4sDl21G5K-F09F8c3i8/edit#heading=h.10ybz1jyaz2)
